token/js/package.json

```diff
@@ -1,13 +1,30 @@
 {
-  "name": "spl-token",
+  "name": "@solana/spl-token",
   "version": "0.0.1",
-  "description": "",
+  "description": "SPL Token Javascript API",
+  "license": "MIT",
+  "author": "Solana Maintainers <maintainers@solana.com>",
+  "homepage": "https://solana.com/",
   "repository": {
     "type": "git",
     "url": "https://github.com/solana-labs/solana-program-library"
   },
+  "bugs": {
+    "url": "https://github.com/solana-labs/solana-program-library/issues"
+  },
+  "publishConfig": {
+    "access": "public"
+  },
+  "main": "client/token.js",
+  "types": "module.d.ts",
+  "files": [
+    "/cli",
+    "/client",
+    "/module.flow.js"
```

Comment:
Oh I think you need to add `/module.d.ts` here as well. I don't think npm is smart enough to include due to the `"types"` field